src/NovelRT/DotNet/RuntimeService.cpp

```diff
@@ -189,23 +189,23 @@ namespace NovelRT::DotNet
         }
     }
 
-    void RuntimeService::freeObject(intptr_t obj)
+    void RuntimeService::freeObject(intptr_t obj) noexcept
     {
         _exports.getActual().FreeObject(obj);
     }
 
-    void RuntimeService::freeString(const char* str)
+    void RuntimeService::freeString(const char* str) noexcept
     {
         _exports.getActual().FreeString(str);
     }
 
-    std::shared_ptr<Ink::InkService> RuntimeService::getInkService()
+    std::shared_ptr<Ink::InkService> RuntimeService::getInkService() noexcept
     {
         return std::make_shared<Ink::InkService>(shared_from_this(), _exports.getActual().GetInkServiceExports);
```

Comment:
Are we sure this is noexcept? Seems like there's a lot going on here and I'm pretty sure some of it can throw.